src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -46,10 +52,18 @@ public ModelManager(ReadOnlyEntryBook addressBook, ReadOnlyUserPrefs userPrefs)
         this.userPrefs = new UserPrefs(userPrefs);
         filteredEntries = new FilteredList<>(versionedAddressBook.getPersonList());
         filteredEntries.addListener(this::ensureSelectedPersonIsValid);
-    }
 
-    public ModelManager() {
-        this(new EntryBook(), new UserPrefs());
+        this.storage = storage;
+
+        // Save the models' address book to storage whenever it is modified.
+        versionedAddressBook.addListener(observable -> {
+            logger.info("Address book modified, saving to file.");
```

Comment:
I feel that this lambda deserves a name to ensure consistency with few lines above

Reply:
Extracted this out to its own method, matching above.